packages/react-button/src/components/Button/useButton.ts

```diff
@@ -46,7 +46,8 @@ export const useButton = (props: ButtonProps, ref: React.Ref<HTMLButtonElement>)
       useARIAButton(props, {
         required: true,
         defaultProps: {
-          ref,
+          // useARIAButton isn't working with React.Ref<HTMLButtonElement | HTMLAnchorElement>
+          ref: ref as React.Ref<HTMLButtonElement>,
```

Comment:
Why is it not working with the other ref?

Reply:
I'll be looking into a solution for this after beta release!